### PR TITLE
JavaScript Path: Add Additional Resource to React Testing Section

### PR DESCRIPTION
**Pull request by justEstif**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**
 
* There were no additional resources suggested, which could provide information and/or preparation for the forthcoming project.

**2. This PR:**

*  Provides a link to a YouTube React Testing playlist that focuses on the applying the testing methods covered in the prior sections. 

**3. Additional Information:**

* My first suggestion was a different playlist, as I said in my initial discord [message](https://discord.com/channels/505093832157691914/540903304046182425/982299384513429545), but I believe that this tutorial's focus on testing, with minimal setup, better fits the rest of the course.
* The use of `fireEvent` rather than `userEvent` is the only difference I noticed.

### CODE REVIEW

File: javascript/react_js/react_testing_part_two.md [hunk 1]
```diff
@@ -157,4 +157,5 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-* It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+* Check out this [React testing library tutorial](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gm4_-5UsNmLqMosM-dzuvQ) for a comprehensive overview the last two lessons.
```
Comment (01zulfi) [inline javascript/react_js/react_testing_part_two.md:160]: I think this is a fine addition. There's a video on Integration Tests in the playlist which is out of scope in the curriculum, we should mention it here.

**Review by 01zulfi — CHANGES_REQUESTED** [review]

File: javascript/react_js/react_testing_part_two.md (hunk 1 above)
Comment (zachmmeyer) [reply javascript/react_js/react_testing_part_two.md:160]: @justEstif were there any particular videos out of this playlist that you were wanting to include? I'm going to give this PR another week.

Comment (justEstif) [reply javascript/react_js/react_testing_part_two.md:160]: I don't think this is necessary anymore.

Comment (zachmmeyer) [reply javascript/react_js/react_testing_part_two.md:160]: @justEstif do you want us to close the ticket?

Comment (justEstif) [reply javascript/react_js/react_testing_part_two.md:160]: Yes.

**Pull request closed without merging**